core/governance/market.go

```diff
@@ -430,6 +448,18 @@ func validateNewMarketChange(
 	return types.ProposalErrorUnspecified, nil
 }
 
+func validateParentMarket(prop *types.NewMarket, parent *types.Market) (types.ProposalError, error) {
+	return types.ProposalErrorUnspecified, nil
```

Comment:
what will we do here? ensure that the details of this market matches the parent or something?

Reply:
yes: settlement assets have to match etc... spec PR with more details is in review currently